src/test/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionDownloaderTest.java

```diff
@@ -172,6 +179,9 @@ void retrievesTranscriptionFromUnstructuredContainer() throws IOException, FileN
 
     @Test
     void throwsNotFoundExceptionIfTranscriptionDocumentHiddenAndUserIsNotSuperAdmin() {
+        when(transcriptionDocumentRepository.findByTranscriptionIdAndHiddenTrueIncludeDeleted(anyInt()))
```

Comment:
Should add a verification to ensure this got called and with the correct value

Reply:
done